.circleci/config.yml

```diff
@@ -130,6 +136,53 @@ commands:
             yarn install --immutable
             cd ..
 
+  server-build:
+    steps:
+      - run:
+          name: "Server: build"
+          command: |
+            cd server
+            yarn build
+            cd ..
+
+  server-start:
+    steps:
+      - run:
+          name: "Server: start"
+          background: true
+          command: |
+            cd server 
+            yarn start
+            cd ..
+
+  ##
+  # Ngrok Commands
+  ##
+  ngrok-install:
+    steps:
+      - run:
+          name: "ngrok: install"
+          command: |
+            wget https://bin.equinox.io/c/bNyj1mQVY4c/ngrok-v3-stable-linux-amd64.tgz
+            sudo tar xvzf ngrok-v3-stable-linux-amd64.tgz
+            sudo chmod +x ngrok
+            ./ngrok authtoken ${NGROK_AUTH_TOKEN}
+      - run:
+          name: "ngrok: run"
+          background: true
+          command: ./ngrok http ${PORT}
```

Comment:
The PORT variable is declared in the pipeline environment and defaults to the same RN service used in Android and iOS parallel jobs?

Reply:
We decided it will be easier to do consecutive jobs due to ngrok only allowing 1 tunnel on free accounts

Reply:
Right. In general it is better to create an ngrok Twilio token for employees there is option to have access to this feature. Then CCI can pick the ngrok token from environment and will be more stable in longer runs. 
This would ensure 3 things:
1. Specify the domain name for endpoint (could ensure that later it can be known for the app). 
2. Removes1 tunnel limitation. But that applies only to the same host. 
3. Tunnel timeouts and max duration until it rotates the endpoint name is also removed for premium. We had problems with random failures to create ngrok tunnel and job timeouts for free tunnels.

Reply:
Yes we will look into upgrading this account to a Twilio Ngrok account in the future